README.md

```diff
@@ -15,20 +15,20 @@ The following sections are editable by making changes to the following files:
 | Use Cases              | Examples for when to use this component           | `./docs/partials/useCases.md`       |
 | Additional Information | For use to add any component specific information | `./docs/partials/readmeAddlInfo.md` |
 | Component Example Code | HTML sample code of the components use            | `./apiExamples/basic.html`          |
--->
-
+-->
+
```

Comment:
This file should not be updated directly. It is generated with the `npm run build:docs` npm script. 

All updates need to be either addressed in the `./docTemplates/README.md` file `./docs/partials` directory or updated in the Auro Generator that @irma-kurnia-alaska is making.

Reply:
I think I did try the npm run build first before git commit to make sure I'm not breaking something.
I will revert back the changes for this file only then.